python/ql/test/query-tests/Variables/unused_local_nonlocal/variables_test.py

```diff
@@ -0,0 +1,36 @@
+
+# FPs involving nonlocal
+
+def nonlocal_fp():
+    test = False
+    def set_test():
+        nonlocal test
+        test = True
+    set_test()
+    if test:
+        print("Test is set.")
+
+nonlocal_fp()
```

Comment:
nitpick: Did you intend to leave the this function call in here?

Reply:
I did. I put it there for the purposes of running the code to double-check the behaviour of the actual Python runtime, but it has no effect on the analysis. Since it's harmless, I figured I might as well leave it in.